prometheus-exporters/cronus-exporter/alerts/cronus.alerts

```diff
@@ -36,43 +36,4 @@ groups:
       playbook: docs/devops/alert/cronus/#maia_down
     annotations:
       description: grafana - https://grafana.{{ $labels.region }}.cloud.sap/d/cronushealth/cronushealth?orgId=1 , slack-cronus-dev - https://convergedcloud.slack.com/archives/G01SLHXERL5
-      summary: cronus maia metrics system is down
-
-- name: NebulaAlerts
-  rules:
-  - alert: NebulaReconcile
-    expr: cronus_simulator_event_test_passed{action="reconcile"} == 0
-    for: 10m
-    labels:
-      meta: Nebula - reconcile account fails in {{ $labels.region }} with message {{ $labels.message }} 
-      service: cronus
-      severity: warning
-      tier: os
-      playbook: docs/devops/alert/cronus/#nebula_reconcile
-    annotations:
-      description: multicloud-api -  https://db.multicloud.int.sap , multicloud-jira - https://jira.multicloud.int.sap , slack-cc-cronus-dev - https://convergedcloud.slack.com/archives/G01SLHXERL5, log - https://sentry.{{ $labels.region }}.cloud.sap/monsoon/nebula/?query=requestID:{{ $labels.request_id }}
-      summary: problem with reconciling an account 
-  - alert: NebulaProductionReady24h
-    expr: sum by (account_id, project_id) (cronus_updater_production_ready{status="active"} offset 24h)  ==  sum by (account_id, project_id) (cronus_updater_production_ready{status="active"}) == 0
-    for: 10m
-    labels:
-      meta: Nebula - the SES accounts are still in Sandbox for more than 24 hours in the region {{ $labels.region }}
-      service: cronus
-      severity: warning
-      tier: os
-      playbook: docs/devops/alert/cronus/#production_ready
-    annotations:
-      description: multicloud-api -  https://db.multicloud.int.sap , multicloud-jira - https://jira.multicloud.int.sap , slack-cc-cronus-dev - https://convergedcloud.slack.com/archives/G01SLHXERL5
-      summary: accounts are still in sandbox after 24 hours
-  - alert: NebulaDeleteNotCompleted
-    expr: (cronus_updater_production_ready{tags=~".*terminated*.", status="active"}  offset 24h) * on(exported_instance, account_id) group_left(tags) cronus_updater_production_ready{tags=~".*terminated*.", status="active"} 
-    for: 10m
-    labels:
-      meta: Nebula - Multicloud fails to delete accounts for more than 24 hours in a region {{ $labels.region }}
-      service: cronus
-      severity: warning
-      tier: os
-      playbook: docs/devops/alert/cronus/#delete_account
-    annotations:
-      description: multicloud-api -  https://db.multicloud.int.sap , multicloud-jira - https://jira.multicloud.int.sap , slack-cc-cronus-dev - https://convergedcloud.slack.com/archives/G01SLHXERL5
-      summary: accounts weren't deleted properly within 24 hours
+      summary: cronus maia metrics system is down
```

Comment:
Git does not like when there is a final new line missing and any diff on this line will be diffed as a full rewrite.